src/Domain/CampaignRepository.php

```diff
@@ -91,27 +91,34 @@ private function pullCharity(
         ?string $stripeAccountId,
         ?string $giftAidOnboardingStatus,
         ?string $hmrcReferenceNumber,
-        ?string $regulator,
```

Comment:
this may as well be non-null because it is being passed to `getRegulatorHMRCIdentifier` anyway, and that already requires a non-null value.